src/main/scala/com/monsanto/arch/cloudformation/model.scala

```diff
@@ -131,4 +133,5 @@ package com.monsanto.arch.cloudformation
 
 package object model {
 
+  implicit def lift2Option[A](a : A) : Option[A] = Option(a)
```

Comment:
This allows not having to wrap arguments in a Some(...). I can remove it if you would rather not have it in there. It seems weird to have to do Some(Seq(...)) as opposed to just Seq(...)